app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/route/RouteFragmentTest.kt

```diff
@@ -80,4 +90,36 @@ class RouteFragmentTest {
     fun searchBarDisplayed(){
         onView(allOf(withId(R.id.search_bar), isAssignableFrom(SearchView::class.java))).check(matches(isDisplayed()))
     }
+
+    @Test
+    fun validSearchShowsAddressSnackBar(){
+        val city = "Lausanne"
+        onView(withId(androidx.appcompat.R.id.search_src_text)).perform(typeText(city)).perform(pressKey(KeyEvent.KEYCODE_ENTER))
+        val geocoder = Geocoder(getApplicationContext())
+        var  result : String
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            geocoder.getFromLocationName(city, 1) { addresses ->
+                val bestAddress = addresses[0]
+                result = bestAddress.toString()
+                onView(withId(com.google.android.material.R.id.snackbar_text)).check(
+                    matches(
+                        withText(result)
+                    )
+                )
+            }
+        } else{
+            val bestAddresses = geocoder.getFromLocationName(city, 1)
+            result = bestAddresses?.get(0).toString()
+            onView(withId(com.google.android.material.R.id.snackbar_text))
+                .check(matches(withText(result)))
+        }
+    }
+
+    @Test
+    fun wrongSearchShowsNoResultSnackbar(){
```

Comment:
This test is not passing for and I have this error :

com.github.factotum_sdp.factotum.ui.route.RouteFragmentTest > wrongSearchShowsNoResultSnackbar[Pixel_6_Pro_API_33(AVD) - 13] FAILED 
        androidx.test.espresso.NoMatchingViewException: No views in hierarchy found matching: view.getId() is <2131231211/com.github.factotum_sdp.factotum:id/snackbar_text>
        If the target view is not part of the view hierarchy, you may need to use Espresso.onData to load it from one of the following AdapterViews:android.widget.ListView{d954126 VFED.VC.. ........ 0,336-1440,1612 #7f08012a app:id/list_view_routes aid=1073741860}
Tests on Pixel_6_Pro_API_33(AVD) - 13 failed: There was 1 failure(s).

Tried two times and same restult